src/main/java/it/smartcommunitylab/aac/webauthn/model/WebAuthnRegistrationResponse.java

```diff
@@ -0,0 +1,63 @@
+package it.smartcommunitylab.aac.webauthn.model;
+
+import java.io.IOException;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+import com.yubico.webauthn.data.PublicKeyCredentialCreationOptions;
+
+@Valid
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class WebAuthnRegistrationResponse {
+
+    @JsonProperty("key")
+    @NotNull
+    private String key;
+
+    @JsonProperty("options")
+    @JsonSerialize(using = PublicKeyCredentialCreationOptionsSerializer.class)
+    @NotNull
+    PublicKeyCredentialCreationOptions options;
```

Comment:
visibility to private

Reply:
Fixed in 91e967badbd4aec94a1635151e57d78fa585a743